packages/react/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # @openfun/cunningham-react
 
+## 0.4.0
+
+### Minor Changes
+
+- 4f11684: Add Field component
+- feea284: Add Input component
+- 43096e2: add Checkbox component
+- f5cb2e7: add Radio component
+
+### Patch Changes
+
+- a462bb5: add compact mode to Field
+- 26be159: make Pagination use new Input
+- Updated dependencies [b9d2f29]
+  - @openfun/cunningham-tokens@0.3.0
```

Comment:
```suggestion
```
Can be removed. I don't know why it adds this into changelog even though we did not add changeset for this change.